### PR TITLE
React warnings fix

### PR DESCRIPTION
**Pull request by pflynn-virtru**
- Fixed React warnings
- Default .env is for Quickstart
- moved craco dependency to devDependency

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1 @@
-REACT_APP_SERVER_DATA={'attributes':'https://opentdf.us/v2/attributes','entitlements':'https://opentdf.us/v2/entitlements','authority':'https://keycloak.opentdf.us/auth/','clientId':'localhost-abacus','access':'https://opentdf.us/kas','realms':'opentdf-realm', 'basePath':''}
\ No newline at end of file
+REACT_APP_SERVER_DATA={'attributes':'http://localhost:3000/v2/attributes','entitlements':'http://localhost:3000/v2/entitlements','authority':'http://localhost:3000/auth/','clientId':'dcr-test','access':'http://localhost:3000/kas','realms':'tdf', 'basePath':''}
```
Comment (elizabethhealy) [inline .env:1]: shouldn't these point to localhost:65432? (assuming quickstart backend)

Comment (pflynn-virtru) [reply .env:1]: yes! goodness

Comment (pflynn-virtru) [reply .env]: updated

File: .env [hunk 2]
```diff
@@ -1 +1 @@
-REACT_APP_SERVER_DATA={'attributes':'https://opentdf.us/v2/attributes','entitlements':'https://opentdf.us/v2/entitlements','authority':'https://keycloak.opentdf.us/auth/','clientId':'localhost-abacus','access':'https://opentdf.us/kas','realms':'opentdf-realm', 'basePath':''}
\ No newline at end of file
+REACT_APP_SERVER_DATA={'attributes':'http://localhost:65432/v2/attributes','entitlements':'http://localhost:65432/v2/entitlements','authority':'http://localhost:65432/auth/','clientId':'dcr-test','access':'http://localhost:65432/kas','realms':'tdf', 'basePath':''}
```
Comment (elizabethhealy) [inline .env:1]: localhost:65432/api/attributes and /api/entitlements?

**Review by elizabethhealy — APPROVED** [review]

File: .env (hunk 2 above)
Comment (pflynn-virtru) [reply .env]: fixed

**Review by elizabethhealy — DISMISSED** [review]

**Pull request merged**